custom-resources/cert-manager-issuer.tf

```diff
@@ -1,7 +1,11 @@
 locals {
-  dns_zones           = try(var.core_services.azure_dns_zones.dns_zones, [])
-  dns_resource_group  = try(var.core_services.azure_dns_zones.resource_group, "")
-  enable_cert_manager = length(local.dns_zones) > 0 && length(local.dns_resource_group) > 0
+  enable_azure_dns = length(var.core_services.azure_dns_zones) > 0
+  zone_split_ids   = local.enable_azure_dns ? [for zone in var.core_services.azure_dns_zones.dns_zone : split("/", zone)] : []
+  azure_dns_zones = local.enable_azure_dns ? { # This is hardcoded to expect a single element. Will need to change this when multiple DNS zones are supported by external DNS.
+    dns_zones      = toset([element(split("/", var.core_services.azure_dns_zones.dns_zone[0]), index(split("/", var.core_services.azure_dns_zones.dns_zone[0]), "dnszones") + 1)])
+    resource_group = element(split("/", var.core_services.azure_dns_zones.dns_zone[0]), index(split("/", var.core_services.azure_dns_zones.dns_zone[0]), "resourceGroups") + 1)
```

Comment:
should this be `resourceGroup`? Is it plural in the string?

Reply:
It's plural in the string:
```
➜  core-services git:(feature-widget-dns) ✗ tf console
> var.core_services.azure_dns_zones.dns_zone
[
  "/subscriptions/xxx-xxx-xxx-xxx/resourceGroups/xxx-xxx/providers/Microsoft.Network/dnszones/xxx.com",
]
>
```